vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerIT.java

```diff
@@ -96,30 +92,6 @@ public void selectDateOnMinMaxDatePicker() {
                 picker.getPropertyBoolean("invalid"));
     }
 
-    @Test
-    public void selectDateOnFinnishDatePicker() {
```

Comment:
Finnish locale is covered in `DatePickerI18nIT` introduced in https://github.com/vaadin/flow-components/pull/1742 so this IT is redundant.